cmd/skopeo/login.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/containers/common/pkg/auth"
+	"github.com/containers/image/v5/pkg/docker/config"
+	"github.com/containers/image/v5/types"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type loginOptions struct {
+	global    *globalOptions
+	loginOpts *auth.LoginOptions
+	getLogin  optionalBool
+	tlsVerify optionalBool
+}
+
+func loginCmd(global *globalOptions) *cobra.Command {
+	opts := loginOptions{
+		global:    global,
+		loginOpts: &auth.LoginOptions{},
+	}
+	cmd := &cobra.Command{
+		Use:     "login",
+		Short:   "Login to a container registry",
+		Long:    "Login to a container registry on a specified server.",
+		RunE:    commandAction(opts.run),
+		Example: `skopeo login quay.io`,
+	}
+	usageHelper(cmd)
+	flags := cmd.Flags()
+	optionalBoolFlag(flags, newOptionalBoolValue(&opts.getLogin), "get-login", "Return the current login user for the registry")
+	optionalBoolFlag(flags, newOptionalBoolValue(&opts.tlsVerify), "tls-verify", "Return the current login user for the registry")
```

Comment:
(This is probably for the other PR: Now that there is an `optionalBoolFlag` helper, it should call `newOptionalBoolValue` internally.)